x-pack/plugins/cloud/public/plugin.ts

```diff
@@ -123,12 +118,21 @@ export class CloudPlugin implements Plugin<CloudSetup> {
    * At this point, we do not have enough information to reliably make this determination,
    * but we do know that all cloud deployment admins are superusers by default.
    */
-  private async checkIfAuthorizedForLinks() {
+  private async checkIfAuthorizedForLinks({
+    http,
+    security,
+  }: {
+    http: HttpStart;
+    security?: SecurityPluginStart;
+  }) {
+    if (http.anonymousPaths.isAnonymous(window.location.pathname)) {
+      return false;
+    }
```

Comment:
(thinking) all our usages of this API are calling it with `window.location.pathname`. We probably should create a shorthand or a default parameter value...

Reply:
Yeah I had the same thought. At some point we might even want an observable...we might transition from an anonymous path to a non-anonymous path without a full page reload